packages/backend/src/boot/master.ts

```diff
@@ -110,8 +107,9 @@ function showNodejsVersion(): void {
 
 	nodejsLogger.info(`Version ${runningNodejsVersion.join('.')}`);
```

Comment:
`runningNodejsVersion` was removed above.

```suggestion
	nodejsLogger.info(`Version ${process.version} detected.`);
```